static/app/views/replays/deadRageClick/deadRageSelectorCards.tsx

```diff
@@ -329,7 +329,7 @@ const LoadingContainer = styled(ContentContainer)`
 `;
 
 const StyledPlaceholder = styled(Placeholder)`
-  height: 34px;
+  height: ${p => (p.theme.isChonk ? '39px' : '34px')};
```

Comment:
this change ensures we don’t get a layout shift when switching from `loading` state to  `data`, as in chonk, the line is now larger in height